upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template

```diff
@@ -291,103 +298,118 @@ spec:
     matchLabels:
       app.kubernetes.io/name: aws-node-termination-handler
       app.kubernetes.io/instance: aws-node-termination-handler
+      app.kubernetes.io/component: daemonset
```

Comment:
I don't think this can be changed:
https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#pod-selector

Reply:
I wonder if we want a presubmit test that upgrade addons from master to PR to catch things like this.

Reply:
Would be nice, but just validation and maybe some conformance. Regular tests are already taking **VERY** long time.

Reply:
The upgrade scenario already allows disabling e2e by setting `KOPS_SKIP_E2E`.